spec/features/assignment_creation_spec.rb

```diff
@@ -214,19 +214,6 @@ def get_selected_id(finder_var)
     before(:each) do
       @assignment = create(:assignment)
       create_list(:participant, 3)
-      # Create an assignment due date
```

Comment:
Actually, I have ever changed other test files. But it seemed not working. To fix this,  I copy the files in Expertiza  to replace the modified files in our pository.